Online-Messaging-App/src/app/home/chatbox/chatbox.component.ts

```diff
@@ -32,19 +35,30 @@ export class ChatboxComponent implements OnInit {
     private _channelId;
 
     get channelId(): any {
+        this.scrollToBottom();
         return this._channelId;
     }
 
     @Input()
     set channelId(value: any) {
         this._channelId = value;
         this.getMessages(this._channelId);
+        this.isNearBottom = false;
     }
 
     ngOnInit(): void {
         this.messagerService.subscribeToSocket().subscribe((data) => {
             if (data.channelId == this.channelId) this.chatMessages.push(data);
```

Comment:
please fix this if statement, please add proper {}

Reply:
Done